en/README.md

```diff
@@ -0,0 +1,1138 @@
+|serverId          |Number     |M|-|A Server ID of the monitoring server.|
+|url               |URI2047    |M|-|A URL of monitoring server. [[Description](#user-content-servertype)]|
+|type              |string     |M|-|A type of monitoring server. [[List](#user-content-servertype)]|
+|nickName          |String255  |M|-|A nickname of monitoring server.|
+|userName          |String255  |M|-|A user name of monitoring server.|
+|password          |String255  |M|-|A pass word of monitoring server.|
+|pollingIntervalSec|Number     |M|-|Polling interval seconds.|
+|retryIntervalSec  |Number     |M|-|Retry interval seconds in case of fail polling.|
+|extendedInfo      |String32767|M|-|Can store your any information.|
+
+```json
+{
+  "id": 1,
+  "result": {
+    "serverId": 1,
+    "url": "http://example.com:80",
+    "type": "12345678-9abc-def0-1234-567891abcdef",
+    "nickName": "exampleName",
+    "userName": "Admin",
+    "password": "examplePass",
+    "pollingIntervalSec": 30,
+    "retryIntervalSec": 10,
+    "extendedInfo": "exampleExtraInfo"
+  },
+  "jsonrpc": "2.0"
+}
+```
+
+### getLastInfo(method)
+
+Can send the diffarence between sent data and having data currently to use got lastInfo.
+
+Such as when start up a plugin, in case of does not saved lastInfo in Hatohol server, get lastInfo as empty string.
+
+***Request(params)***
+
+|params object value|Type |M/O|Default value|Description|
+|:---------------------|:--|:-:|:----------:|:---|
+|Designate element|String255|M|-|Need to select in the following elements list.|
+
+|Value list|Description|
+|:-------------------------|:---|
+|"host"               |Select latest host information.|
+|"hostGroup"          |Select latest host group information.|
+|"hostGroupMembership"|Select latest host group membership information.|
+|"trigger"            |Select latest trigger information.|
+|"event"              |Select latest event information.|
+|"hostParent"         |Select latest host parent information.|
+
+```json
+{
+  "id": 1,
+  "params": "trigger",
+  "method": "getLastInfo",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+|Object value|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|latest information|String255|M|-|Designate element latest information of to saved in Hatohol server.|
+
+```json
+{
+  "id": 1,
+  "result": "201504011349",
+  "jsonrpc": "2.0"
+}
+```
+In this example, receive a timestamp as lastInfo.
+
+### putItems(method)
+
+Standard motion is to send all item information to Hatohol server when complate conect to Hatohol server and get [fetchItems](#user-content-fetchitemsmethod) request from Hatohol server. It has possible that burden to Hatohol server, can not use anytime. 
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|items  |object   |M|-|This is object that stores item information. Look at the following table for details.|
```

Comment:
Look at the following table for details. ->
In more detail, please refer following table.